app/src/components/results/Results.js

```diff
@@ -87,6 +87,7 @@ class Results extends Component {
   //TODO:DM - should we extract table construction into ResultList comp?
   render() {
     const { classes, query } = this.props;
+    const { selectedIdx } = this.state;
```

Comment:
@arkxu, great! thanks for adding this important and fortunately simple feature.

what do you think re: the practice of "importing" prop and state values at the head of a render fn?

I've seen that a few times, but am somewhat leaning away from the practice myself.

I usually find the code easier to understand when the values being used are fully "pathed" (ex: `this.props.classes` rather than just `classes`) tying back to it being an instance value and then from props/state/etc.

let me know if you have a personal pref or saw a clear edict somewhere in community dox

Reply:
@davemasselink there is one eslint rule about it: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md

I haven't dive deep on why this rule exists but better to follow now as we may enable eslint starting from sometime :)

Reply:
great, that's all I need to see... i'll take a quick look at working linting into build, if not already... or incorporating this rule if there's already a step there